templates/install/install.tpl

```diff
@@ -15,6 +15,11 @@
 		{translate key="installer.appInstallation"}
 	</h1>
 
+	<notification>
+		{translate key="installer.updatingInstructions" upgradeUrl=$upgradeUrl}
```

Comment:
The `$upgradeUrl` is empty here because this is defined [later in the template](https://github.com/pkp/pkp-lib/blob/master/templates/install/install.tpl#L37). Move that `{capture}` up above and it should fix it.